tests/gui/conversation/export/test_device.py

```diff
@@ -192,7 +192,7 @@ def test_Device_print_file_file_missing_not_qubes(homedir, mocker, session):
 
 def test_Device_print_file_when_orig_file_already_exists(homedir, config, mocker, source):
     """
-    The signal `print_requested` should still be emmited if the original file already exists.
+    The signal `print_requested` should still be emited if the original file already exists.
     """
```

Comment:
(dkm but if we are fixing typos: `emitted`. Not worth changing on its own, only if there are other changes to make as well.)

Reply:
:facepalm: I was _changing_ the typo, not fixing it! Thank you, I'll fix that.

Reply:
I forgot fixing this one, I'll leave the comment open until I do.